SetupDataPkg/ConfProfileMgrDxe/ConfProfileMgrDxe.c

```diff
@@ -298,12 +309,7 @@ ConfProfileMgrDxeEntry (
   // ValidateProfile does not return a status, in case of failure, it writes the chosen profile to flash
   // and resets the system. Only validate the profile if we are in CUSTOMER_MODE (not manufacturing mode),
   // as in debug and bringup scenarios the profile may be expected to not match

```

Comment:
Can this comment be updated to reflect that we will still validate the size and attributes of the flash variables in manufacturing mode, but only validate the data in customer mode?